hpOneView/resources/settings/restores.py

```diff
@@ -35,7 +35,8 @@ class Restores(object):
 
     DEFAULT_VALUES = {
         '200': {"type": "RESTORE"},
-        '300': {"type": "RESTORE"}
+        '300': {"type": "RESTORE"},
+        '1200': {"type": "RESTOREV1000"}
     }
```

Comment:
Since this PR is raised only for API 1200, this seems ok for me. @sijeesh, Can you please confirm.